docs/markdown/Unit-tests.md

```diff
@@ -234,6 +234,9 @@ other useful information as the environmental variables. This is
 useful, for example, when you run the tests on Travis-CI, Jenkins and
 the like.
 
+### Timeout
+To disable timeout in test cases, add `timeout: 0` or negative value to allow infinite duration for test case to completes. Alternatively, `timeout_multiplier: 0` accomplish the same functionality as `timeout: 0`. For running test, you can specify command line argument for overriding timeout as well by supplying `--timeout-multiplier 0` argument. The `timeout` option of test case is specified in a number of seconds where 60 seconds equals to 1 minute and go on. The `timeout-multiplier`multiplies the specified timeout duration by a given factors for the timeout-multiplier option.
```

Comment:
> [...] and go on.

Should that be "so on" instead? To indicate that it's 60min in 1 hour etc.

> [...] 60 seconds equals to 1 minute [...]

shouldn't it be either "60 seconds equal 1 minute" or "60 seconds are equal to 1 minute"

Reply:
Honestly, this feels like giving people lessons in units of time measurement.

Reply:
I'll remove it.